cmd/api-mock/main.go

```diff
@@ -166,6 +168,28 @@ func (a *AddonStatusCreateEndpoint) ServeHTTP(w http.ResponseWriter, r *http.Req
 			fmt.Fprintln(w, `{}`)
 			return
 		}
+
+		if apiMockConfig.FailOnAddonStatusCreateEndpoint == "true" {
```

Comment:
@pepedocs cant we execute this from the configure function itself ? or it doesnt matter ?

Reply:
@venkateshsredhat  Nope, those flags are to modify the behvior of the server on-the-fly so it has to be in the handler.